tests/backup/backup_basic_test.go

```diff
@@ -40,12 +39,8 @@ var (
 	wantAfterSuiteSystemCheck     bool = false
 	wantAfterSuiteValidateCleanup bool = false
 	// User should keep updating the below 3 datas
-	pre_rule_app   = []string{"cassandra", "postgres"}
-	post_rule_app  = []string{"cassandra"}
-	app_parameters = map[string]map[string]string{
-		"cassandra": {"pre_action_list": "nodetool flush -- keyspace1;", "post_action_list": "nodetool verify -- keyspace1;", "background": "false", "run_in_single_pod": "false"},
-		"postgres":  {"pre_action_list": "PGPASSWORD=$POSTGRES_PASSWORD; psql -U \"$POSTGRES_USER\" -c \"CHECKPOINT\";", "background": "false", "run_in_single_pod": "false"},
-	}
+	pre_rule_app  = []string{"cassandra", "postgres"}
+	post_rule_app = []string{"cassandra"}
```

Comment:
Change `pre_rule_app` and `post_rule_app` to camel case

Reply:
done!!